crates/ide_assists/src/handlers/fill_match_arms.rs

```diff
@@ -481,6 +492,67 @@ fn main() {
                 }
             }
             "#,
+            r#"
+            enum A { One, Two }
+            enum B { One, Two }
+
+            fn main() {
+                let a = A::One;
+                let b = B::One;
+                match (a, b) {
+                    (A::Two, B::One) => {}
+                    $0(A::One, B::One) => {}
+                    (A::One, B::Two) => {}
+                    (A::Two, B::Two) => {}
+                }
+            }
+            "#,
+        );
+    }
```

Comment:
If we are touching this test anyway, lets unindent it as per our style guide.

Reply:
Done.